sycl/include/CL/sycl/detail/rocm_definitions.hpp

```diff
@@ -0,0 +1,24 @@
+//==------------ cuda_definitions.hpp - SYCL CUDA backend ------------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+// CUDA backend specific options
+// TODO: Use values that won't overlap with others
+
+// Mem Object info: Retrieve the raw CUDA pointer from a cl_mem
+#define __SYCL_PI_HIP_RAW_POINTER (0xFF01)
+// Context creation: Use a primary CUDA context instead of a custom one by
+//                   providing a property value of PI_TRUE for the following
+//                   property ID.
+#define __SYCL_PI_CONTEXT_PROPERTIES_HIP_PRIMARY (0xFF02)
+
+// PI Command Queue using Default stream
+#define __SYCL_PI_HIP_USE_DEFAULT_STREAM (0xFF03)
+// PI Command queue will sync with default stream
+#define __SYCL_PI_HIP_SYNC_WITH_DEFAULT (0xFF04)
```

Comment:
Nit. Is there any reason not to use constexpr variables here?

Reply:
We refer to the code of cuda_definitions.hpp, so it looks consistent.